infrastructure/load-balancers.yaml

```diff
@@ -20,6 +22,18 @@ Parameters:
         Description: Select the Security Group to apply to the Applicaion Load Balancer
         Type: AWS::EC2::SecurityGroup::Id
 
+    PublicAlbAcmCertificate:
+        AllowedPattern: ^$|(arn:aws:acm:)([a-z0-9/:-])*([a-z0-9])$
+        Description: '[ Optional ] The AWS Certification Manager certificate ARN for the ALB certificate - this certificate should be created in the region you wish to run the ALB and must reference the domain name you use below.'
+        Type: String
+
+Conditions:
+
+    SslCertificate:
+        !Not [!Equals [ '', !Ref PublicAlbAcmCertificate ] ]
+    NoSslCertificate:
+        !Equals [ '', !Ref PublicAlbAcmCertificate ]
```

Comment:
So if a value for `PublicAlbAcmCertificate` is _not_ provided, then the `SslCertificate` condition is false and the `NoSslCertificate` condition is true.